app/utils.py

```diff
@@ -35,6 +35,18 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # CONTENT
 
+# ----------------------------------------------------------------------------------------------------------------------
+# User Info
+# The following block is used for the definition of logged user informations
+
+
+def user_department_lat_lon():
```

Comment:
I see where we use this function but as mentioned in the related comment, I am not sure whether it is useful.